### DISCUSSION TITLE
What must i understand with run results?

### DISCUSSION THREAD

**Discussion by mertdogan** [BrainJS/brain.js#718] [Q&A] [answered]
I trained a model with a dataset by net = new brain.NeuralNetwork(); and then run with prediction data.

X: 1.9106081161623933e-9
Y: 0.9961457252502441
Z: 0.9999985098838806

Prediction results as above. As you can see, both Y and Z are nearly 1. What must i understand with this results?

**Comment by robertleeplummerjr** [comment]
What is the training data?

**Reply by mertdogan** [reply]
Training data has alot of items. A sample as below:

input:
ALTLIGID: 533
C: 1.67
CS12: 1.07
CSX2: 2.25
DGYE: 4.1
DGYH: 1.05
DGYKOLUR: 9.4
ESGYE: 1.65
ESGYH: 1.68
GAT1: 1.78
GAT2: 9.4
GAT12: 1.82
GATX: 11
IG1: 1.22
IG2: 3
IGO: 11
IYMS1X: 17
IYMS2X: 17
IYMS11: 1.6
IYMS12: 80
IYMS21: 19
IYMS22: 11
IYMSX1: 2.95
IYMSX2: 13
IYMSXX: 5.4
KGVAR: 1.82
KGYOK: 1.52
KODCS: 31683
KODDGY: 31738
KODDGYK: 31733
KODESGY: 31737
KODGAT: 31740
KODIG: 31687
KODIYMS: 31727
KODKG: 31720
KODMS: 31682
KODSK: 31729
KODTC: 31742
LIGID: 68
MB: 3
MID: 90065
MS1: 1.22
MS2: 6.2
MSX: 4
SEZONID: 599
SK00: 11
SK01: 16
SK02: 45
SK03: 200
SK04: 250
SK05: 250
SK06: 250
SK10: 5.2
SK11: 8
SK12: 21
SK13: 90
SK14: 250
SK15: 250
SK20: 4.8
SK21: 6.8
SK22: 19
SK23: 80
SK24: 250
SK30: 6.4
SK31: 9.2
SK32: 26
SK33: 120
SK40: 11
SK41: 16
SK42: 45
SK50: 25
SK51: 35
SK60: 70
T: 1.67
T1I: 1549
T2I: 6583
TIP: 1

output:
DURUMMS: X

**Reply by mertdogan** [reply]
If you want me to share full data, i can share.

**Reply by mertdogan** [reply]
> What is the training data?

May i wait for reply or not?

**Comment by mertdogan** [comment]
Hi Mr. Robert. Is this results normal or not?

**Comment by robertleeplummerjr** [comment, marked as answer]
You'll want to normalize your data on the same scale.  Likely between 0 and 1 for each value.  You can later de-normalize it (or scale it).  The problem you're going to run into is that a input that is very tiny is like a whisper into your ear, and a large number is like yelling.  By putting all your data in a scale between 0 and 1 it allows the network to better understand what's going on.  This is a principle throughout machine learning.
